src/keybind.cpp

```diff
@@ -1252,13 +1252,15 @@ void	kf_ToggleGodMode()
 		}
 		// remove all proximity messages
 		releaseAllProxDisp();
+		radarPermitted = false; //remove minimap
```

Comment:
You will want to use `radarPermitted = structureExists(selectedPlayer, REF_HQ, true, false) || structureExists(selectedPlayer, REF_HQ, true, true)` here as this currently will knock out the minimap if the player has a HQ and uses the deity cheat.

Reply:
This ok. But still need somehow edit rules.js, couse any changes on the player base - removes minimap.
But this is all debugging, not a game, I think it's not important